Makefile

```diff
@@ -238,3 +238,7 @@ postgres:
 
 verify: tidy generate
 	git diff --exit-code
+	
+.PHONY: generate-report
+generate-report: manifests generate fmt vet ## Run a controller from your host.
+	go run cmd/reporting-operator/main.go start
```

Comment:
Introducing this kind of Makefile target is reasonable to me but I'd like to see it in a slightly different form:

- Can we introduce a bin/server target that's responsible for building the server main.go package and outputting a bin/server binary.
- Add a .PHONY target for that bin/server binary so we can rebuild that binary if there's already an existing bin/server executable sitting locally

I think this would look like the following:

```makefile
.PHONY: bin/server
bin/server:
    go build -o $@ main.go
```

Users would then run `./bin/server start ...` directly without interacting with go run.